langchain4j/src/main/java/dev/langchain4j/service/TokenStream.java

```diff
@@ -87,6 +89,40 @@ default TokenStream onPartialThinkingWithContext(BiConsumer<PartialThinking, Par
         throw new UnsupportedOperationException("not implemented");
     }
 
+    /**
+     * The provided consumer will be invoked every time a new partial tool call
+     * (usually containing a single token of the tool's arguments) from a language model is available.
+     * <p>
+     * Either this or the {@link #onPartialToolCall(BiConsumer)} callback can be used
+     * if you want to consume partial tool calls as soon as they become available.
+     *
+     * @param partialToolCallHandler lambda that will be invoked when a model generates a new partial tool call
+     * @return token stream instance used to configure or start stream processing
+     * @see #onPartialToolCall(BiConsumer)
+     * @since 1.12.0
```

Comment:
```suggestion
     * @since 1.11.0
```
Please fix this everywhere